dags/copy_deduplicate.py

```diff
@@ -75,16 +87,24 @@
         target_project_id="moz-fx-data-shared-prod",
         billing_projects=("moz-fx-data-shared-prod",),
         only_tables=["telemetry_live.main_v4"],
-        parallelism=24,
-        slices=100,
+        priority_weight=100,
+        parallelism=1,
         owner="jklukas@mozilla.com",
         email=[
             "telemetry-alerts@mozilla.com",
             "relud@mozilla.com",
             "jklukas@mozilla.com",
         ],
+    )
+
+    copy_deduplicate_event_ping = bigquery_etl_copy_deduplicate(
```

Comment:
I see no dependencies on `event_v4` in bigquery-etl (except for views on top of the live table) since the derived event tables are more appropriate for all downstream analysis, so I think we are safe to split out the event ping to its own task without having to make coordinated changes outside the scope of this DAG.